zookeeper-server/src/test/java/org/apache/zookeeper/server/X509AuthFailureTest.java

```diff
@@ -0,0 +1,242 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zookeeper.server;
+
+//import com.salesforce.sds.keystore.DynamicKeyStoreProvider;
+import org.apache.zookeeper.PortAssignment;
+import org.apache.zookeeper.ZooKeeper;
+import org.apache.zookeeper.client.ZKClientConfig;
+import org.apache.zookeeper.common.ClientX509Util;
+import org.apache.zookeeper.common.PathUtils;
+import org.apache.zookeeper.server.admin.JettyAdminServer;
+import org.apache.zookeeper.server.quorum.QuorumPeerTestBase;
+import org.apache.zookeeper.test.ClientBase;
+import org.junit.After;
+import org.junit.Before;
```

Comment:
I think we are now using `org.junit.jupiter.api` - so for example here should be `org.junit.jupiter.api.BeforeEach`  and the test api would be `org.junit.jupiter.api.Test`

Reply:
@hanm the "juipiter" is being used in master, not in branch 3.6.. we need this change in branch 3.6 as sfdc is planning to use it.
I can create another PR against master with this change and add the Jupiter changes there

Reply:
https://github.com/apache/zookeeper/pull/1523

Reply:
yeah I didn't realize this PR was targeting branch-3.6. So it's fine here. 

On a side note the usual workflow I believe is to submit pr to master branch first, then after master branch merges, backport the change to other branches (unless, the change is only applicable to other branch like version specific bug fixes).